scripts/irods/test/test_experimental_parallel_replication.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import print_function
+import sys
+if sys.version_info >= (2, 7):
+    import unittest
+else:
+    import unittest2 as unittest
+import os
+from .. import lib
+from . import session
+
+class TestParallelReplication(session.make_sessions_mixin([], []), unittest.TestCase):
+
+    def setUp(self):
+        super(TestParallelReplication, self).setUp()
+        with session.make_session_for_existing_admin() as admin:
+             admin.assert_icommand(['iadmin', "mkresc", "testResc", 'unixfilesystem', lib.get_hostname() + ":" + "/tmp/irods/testResc"], 'STDOUT_SINGLELINE', 'unixfilesystem')
+             admin.assert_icommand(['iadmin', "mkresc", "anotherResc", 'unixfilesystem', lib.get_hostname() + ":" + "/tmp/irods/anotherResc"], 'STDOUT_SINGLELINE', 'unixfilesystem')
+             admin.assert_icommand(['iadmin', "mkuser", "alice", 'rodsuser'])
```

Comment:
i think these are already created by the test harness... surprised you didn't hit duplicate entries...

Reply:
I am not inheriting from the resource base class so I had to make them myself.